.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict = true
```

Comment:
Since engines is already defined in package.json, what do you think of removing the .nvmrc file?

Reply:
Fine by me

Reply:
ceb943a